Libplanet.Explorer.Executable/Options.cs

```diff
@@ -85,6 +85,24 @@ public class Options
             HelpText = "The name of MySQL database to use.")]
         public string MySQLDatabase { get; set; }
 
+        [Option(
+            "max-transactions-per-block",
+            Default = 100,
+            HelpText = "The number of maximum transactions able to be included in a block.")]
+        public int MaxTransactionsPerBlock { get; set; }
+
+        [Option(
+            "max-block-bytes",
+            Default = 100 * 1024,
+            HelpText = "The number of maximum bytes size of blocks except for genesis block.")]
+        public int MaxBlockBytes { get; set; }
+
+        [Option(
+            "max-genesis-bytes",
+            Default = 1024 * 1024,
```

Comment:
Size of the genesis block used in mainnet is 16MB, so how about rescale default value to it?

Reply:
Honestly, I didn't think so. I tried to follow the default value of `Libplanet.Blockchain.Policies.BlockPolicy<T>.ctor`'s `maxGenesisBytes` parameter because libplanet-explorer is not only for the 9c mainnet. 🤔